mm2src/coins/lp_coins.rs

```diff
@@ -2152,39 +2158,110 @@ impl CoinsContext {
         })))
     }
 
-    pub async fn add_coin(&self, coin: MmCoinEnum) -> Result<(), MmError<CoinIsAlreadyActivatedErr>> {
+    pub async fn add_token(&self, coin: MmCoinEnum) -> Result<(), MmError<RegisterCoinError>> {
         let mut coins = self.coins.lock().await;
         if coins.contains_key(coin.ticker()) {
-            return MmError::err(CoinIsAlreadyActivatedErr {
-                ticker: coin.ticker().into(),
+            return MmError::err(RegisterCoinError::CoinIsInitializedAlready {
+                coin: coin.ticker().into(),
             });
         }
+        let ticker = coin.ticker();
+
+        let mut platform_coin_tokens = self.platform_coin_tokens.lock();
+        // Here, we tried to add to a token to platform_coin_tokens if the token belongs to a platform coin.
```

Comment:
Typo, should be `// Here, we try to add a token to platform_coin_tokens if the token belongs to a platform coin.`